packages/bpk-component-map/src/BpkMapMarker.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import { OverlayView } from 'react-google-maps';
+
+export default OverlayView;
```

Comment:
I'm actually not sure that this should exist at all now. The way you use `OverlayView` is like this:

```
<OverlayView
  position={{ lat: 55.944357, lng: -3.1967116 }} 
  mapPaneName="overlayMouseTarget"
>
  // Content goes here, e.g a BpkIcon or a popover or whatever.
</OverlayView>
```

As things stand, consumers will need to pass in children to this component to have it do anything.

So, maybe we should shelve this until we can provide more value. For example, if we define some marker types, we could provide an API like this:

```
<BpkMapMarker
  position={{ lat: 55.944357, lng: -3.1967116 }} 
  type={MARKER_TYPES.ATTRACTION}
  onClick={() => {}}
/>
```

If we leave this as-is, then change it later, we'll have to make breaking changes, which is why I'm leaning towards leaving it for now..